static/app/components/surfaceProvider.tsx

```diff
@@ -4,13 +4,15 @@ export const SurfaceContext = createContext<string>('');
 
 export default function SurfaceProvider({
   children,
-  suffix,
+  value,
+  overrideParent = false,
 }: {
   children: ReactNode;
-  suffix: string;
+  value: string;
+  overrideParent?: boolean;
 }) {
-  const parent = useContext(SurfaceContext);
-  const surface = parent ? `${parent}.${suffix}` : suffix;
+  const parentSurface = useContext(SurfaceContext);
+  const surface = overrideParent || !parentSurface ? value : `${parentSurface}.${value}`;
```

Comment:
could we add a test where `overrideParent` is set?

Reply:
Should I make this in a surfaceProvider.spec test in the same folder?

Reply:
sure, that works